app/controllers/publishers_controller.rb

```diff
@@ -56,7 +56,7 @@ def create
       return redirect_to sign_up_publishers_path
     end
 
-    @publisher = Publisher.new(pending_email: email)
+    @publisher = Publisher.find_or_create_by(pending_email: email, email: nil)
```

Comment:
Can we add a comment explaining why we're not just doing
```ruby
Publisher.find_or_create_by(pending_email: email)
```
Something along the lines of
```
    # Requiring `email: nil` ensures we do not select a publisher with the same pending_email
    # as a publisher in the middle of the change email flow
```

Reply:
This might cause a double create in the future. Since the values for `pending_email` and `email` are the same, shouldn't we have a flag on whether or not the email is verified? Let's talk about it more in person.